### DISCUSSION TITLE
Does AckingWirelessInterface send acknowledgements?

### DISCUSSION THREAD

**Discussion by arahman33** [omnetpp/omnetpp#1162] [General]
I am trying out the tutorials of INET using "AckingWireless Interface"and when I set *.*.wlan[0].mac.useAck = true, I was hoping to see a transmission from the destination to the source but I don't. I am missing out something?

**Comment by levy** [comment]
The ack is implemented with a direct C++ method call from the receiver to the sender. There's no ACK packet sent between them.
